client/src/components/MultiJurisdictionAudit/RoundManagement/OfflineBatchRoundDataEntry.tsx

```diff
@@ -128,12 +128,18 @@ const OfflineBatchRoundDataEntry = ({ round }: IProps) => {
   const { results, finalizedAt } = batchResults
 
   const total = (choiceId: string) =>
-    sum(results.map(batch => batch.choiceResults[choiceId]))
+    sum(results.map(batch => Number(batch.choiceResults[choiceId])))
+
+  // set default choiceResults to blank for controlled input
+  const defaultChoiceArr: { [key: string]: string } = {}
+  contest.choices.forEach(choice => {
+    defaultChoiceArr[choice.id] = ''
+  })
```

Comment:
dang, this is annoying. i did a little research and i think you're right that passing an empty string is the correct solution (https://github.com/facebook/react/issues/7779)

here's a way to do it that has a smaller footprint:

```
diff --git a/client/src/components/MultiJurisdictionAudit/RoundManagement/OfflineBatchRoundDataEntry.tsx b/client/src/components/MultiJurisdictionAudit/RoundManagement/OfflineBatchRoundDataEntry.tsx
index 6861c5a2..2c16a76e 100644
--- a/client/src/components/MultiJurisdictionAudit/RoundManagement/OfflineBatchRoundDataEntry.tsx
+++ b/client/src/components/MultiJurisdictionAudit/RoundManagement/OfflineBatchRoundDataEntry.tsx
@@ -83,6 +83,7 @@ const InputWithValidation = ({ field, form, ...props }: FieldProps) => {
         className={`bp3-input bp3-fill ${error ? 'bp3-intent-danger' : ''}`}
         {...field}
         {...props}
+        value={field.value || ''}
       />
     </div>
   )
```